src/webgl/p5.RendererGL.Retained.js

```diff
@@ -50,32 +69,41 @@ p5.RendererGL.prototype.createBuffers = function(gId, obj) {
     shader = this.setShader(this._getColorShader());
   }
 
-  // allocate space for vertex positions
-  var data = new Float32Array(vToNArray(obj.vertices));
+  /****THIS IS HOW VERTICES WERE DRAWN BEFORE****/
+
+  // // allocate space for vertex positions
+  // var data = new Float32Array(vToNArray(obj.vertices));
+  // shader.enableAttrib(shader.attributes.aPosition.location,
+  //   3, gl.FLOAT, false, 0, 0);
+  // gl.bindBuffer(gl.ARRAY_BUFFER, this.gHash[gId].vertexBuffer);
+  // gl.bufferData(gl.ARRAY_BUFFER, data, gl.STATIC_DRAW);
+
+  var data = new Float32Array(flatten(flatten(obj.lineVertices)));
   shader.enableAttrib(shader.attributes.aPosition.location,
     3, gl.FLOAT, false, 0, 0);
-  gl.bindBuffer(gl.ARRAY_BUFFER, this.gHash[gId].vertexBuffer);
+  gl.bindBuffer(gl.ARRAY_BUFFER, this.gHash[gId].lineVertexBuffer);
   gl.bufferData(gl.ARRAY_BUFFER, data, gl.STATIC_DRAW);
```

Comment:
One more related comment: I've been thinking about if the prep for the buffers could be abstracted enough to go into p5.Renderer.GL instead of in the separate immediate mode and retain mode files. I'm not yet convinced that the answer is yes, or that it's a good thing to do. I'm curious if you have any thoughts on this!